src/main/target/ALIENFLIGHTF3/target.h

```diff
@@ -67,6 +67,7 @@
 #define USE_MAG
 #define USE_MAG_AK8963
 #define MAG_AK8963_ALIGN        CW180_DEG_FLIP
+#define USE_MAG_LIS3MDL
```

Comment:
Most F3 targets are permanently at the brink of overflowing the flash space (a fact that is hidden at the moment by LED_STRIP having been disabled during the development phase to avoid the need for almost daily feature removals). For this reason, adding this driver to F3 targets is problematic. Can you please remove this driver from all F3 based targets?

Reply:
No problem.

This situation makes me think that it would be nice if there was a standardized document for each target, that explained it's features like available solder pads and pinouts.

That way I could have added the driver to only boards that have I2C solder pads initially.